src/styles/components/_hamburger.scss

```diff
@@ -0,0 +1,129 @@
+.hamburger {
+  position: absolute;
+  width: 100%;
+  user-select: none;
+
+  @include respond-above('md') {
+    display: none;
+  }
+
+  > div > span {
+    position: relative;
+    display: block;
+    width: rem(24px);
+    height: rem(3px);
+    border-radius: rem(3px);
+    opacity: 1;
+    margin-bottom: rem(5px);
+
+    background: $dark;
+    transform: rotate(0deg);
+
+    @include transition(transform, 0.25s, ease-in-out);
+  }
+
+  & span:nth-child(1) {
+    transform-origin: left center;
+  }
+
+  & span:nth-child(2) {
+    width: rem(16px);
+    transform-origin: left center;
+  }
+
+  & span:nth-child(3) {
+    margin: 0;
+    transform-origin: left center;
+  }
+
+  &__icon {
+    box-sizing: border-box;
+    padding-left: rem(16px);
+  }
+
+  &__input {
+    position: absolute;
+    z-index: $zindex-hamburger-input;
+    left: rem(16px);
+
+    width: rem(24px);
+    height: 100%;
+    opacity: 0;
+    margin: 0;
+
+    cursor: pointer;
+    transform: rotate(0deg);
+
+    @include transition(transform, 0.55s, ease-in-out);
+
+
+    &:checked {
+      width: rem(19px);
+      height: 100%;
+    }
+
+    &:checked ~ .hamburger__menu {
+      opacity: 1;
+      pointer-events: visible;
+    }
+
+    &:checked ~ div > span {
+      opacity: 1;
+    }
+
+    &:checked ~ div > span:nth-child(1) {
+      top: -1px;
+      left: 1px;
+      transform: rotate(45deg);
+    }
+
+    &:checked ~ div > span:nth-child(2) {
+      opacity: 0;
+      transform: rotate(0deg);
+    }
+
+    &:checked ~ div > span:nth-child(3) {
+      top: 0;
+      left: 1px;
+      transform: rotate(-45deg);
+    }
+  }
+
+  &__menu {
+    position: absolute;
+
+    overflow: hidden;
```

Comment:
Quick question, do we really need this property?

Reply:
No need krub.